src/components/include/cos_asm_server_stub_simple_stack.h

```diff
@@ -32,8 +32,9 @@ name##_inv:                        \
 	xor %ebp, %ebp;            \
 	pushl %edi;                \
 	pushl %esi;                \
+	pushl %ebx;                \
 	call name ;                \
-	addl $16, %esp;            \
+	addl $20, %esp;            \
```

Comment:
@phanikishoreg 

I'd like your eyes on this as well.  I'm not sure I can evaluate this one on my own.